src/main/webapp/app/pages/newsPage/NewsPageContent.tsx

```diff
@@ -180,7 +180,7 @@ export const NEWS_BY_DATE: { [date: string]: NewsData } = {
       'Changed annotation and addition of therapies for variants with a level of evidence',
     changedAnnotation: [
       [
-        'ESR1',
```

Comment:
I would not recommend to modify the data directly. If you look closely how `changedAnnotation` info is rendered in the NewsList, I would expect you to attach GenePageLink in the data that feeds into ChangedAnnotationListItem